Kaggle Connector Update (#103)
googledatastudio/community-connectors: kaggle/README.md

Click [here][managed-deployment] to try the managed deployment.

## Prerequisites for using Kaggle connector

1. Sign up for a [Kaggle][kaggle] account.
1. Go to the 'Account' tab of your user profile
  (https://www.kaggle.com/{username}/account) and select 'Create API Token'.
  This will trigger the download of kaggle.json, a file containing your API
  credentials.

   **Note: Kaggle requires username and password for authentication which can
   be found in the downloaded kaggle.json file. The username should be used as
   it is whereas the "key" should be used as password for authentication.**

## Limitations

> - Currently, only CSV files of upto 20MB size are supported.

Reviewer: ```suggestion
- Currently, only CSV files up to 20MB size are supported.
```
